src/ssl.c

```diff
@@ -31297,6 +31318,75 @@ int SetDsaInternal(WOLFSSL_DSA* dsa)
 #endif /* !NO_DSA && OPENSSL_EXTRA */
 
 
+#if defined(HAVE_ED25519) && defined(OPENSSL_EXTRA)
+static void InitwolfSSL_ED25519(WOLFSSL_ED25519* ed25519)
+{
+    if (ed25519) {
+        ed25519->p        = NULL;
+        ed25519->k        = NULL;
+        ed25519->internal = NULL;
+        ed25519->inSet    = 0;
+        ed25519->exSet    = 0;
```

Comment:
I prefer to use a `XMEMSET`. Makes it easier to add fields and not break anything.

Reply:
I saw both styles used and opted for this. I can change it.